[ci-app] Update gh comments docs (#15562)
DataDog/documentation: content/en/continuous_integration/guides/pull_request_comments.md

1. Go to the [GitHub App integration tile][2] and install the [Datadog GitHub App][1].
2. Give the application read and write permissions for pull requests.

> In the UI, you can do it in two different ways:

Reviewer: ```suggestion
In the UI, you can do this from the settings page or the commit or branch page.
```

> 

Reviewer: ```suggestion

#### CI Settings page

```

1. Go to [CI Test Service Settings page][3] and search for the repository or test service.
2. Toggle the `GitHub Comments` column for the desired service.

{{< img src="ci/enable-settings-github-comments.png" alt="Enable GitHub comments dropdown" style="width:100%;">}}

> Or

Reviewer: ```suggestion
#### Commit or branch page
```
